src/core/model_repository_manager.cc

```diff
@@ -835,24 +838,28 @@ ModelRepositoryManager::Create(
   std::unique_ptr<ModelRepositoryManager> local_manager(
       new ModelRepositoryManager(
           status_manager, repository_path, backend_config_map,
-          !strict_model_config, polling_enabled, std::move(life_cycle)));
+          !strict_model_config, polling_enabled, model_control_enabled,
+          std::move(life_cycle)));
 
   // Similar to PollAndUpdate(), but simplier
   std::set<std::string> added, deleted, modified, unmodified;
-  if (polling_enabled) {
+  if (!model_control_enabled) {
```

Comment:
Don't want to allow polling if control mode is NONE.

Reply:
This is the poll for startup, so always poll as long as the mode is not EXPLICIT